File: platforms/web/examples/scene/material/metallic_roughness_texture.js
import { Material, Renderer } from "fragmentcolor";

const renderer = new Renderer();
const mr_map_bytes = [ 0, 200, 50, 255, 0,   240, 80, 255, 0,   180, 30, 255, 0,   220, 60, 255, ];
const mr_map = await renderer.createTexture(mr_map_bytes, [2, 2]);
const mat = Material.pbr().metallicRoughnessTexture(mr_map);

File: platforms/swift/examples/scene/material/metallic_roughness_texture.swift
import FragmentColor

let renderer = Renderer()
let mr_map_bytes = [
    0, 200, 50, 255,
    0,   240, 80, 255,
    0,   180, 30, 255,
    0,   220, 60, 255,
]
let mr_map = try await renderer.createTexture((mr_map_bytes, [2, 2]))
let mat = Material.pbr().metallicRoughnessTexture(mr_map)